pages/collection.html

```diff
@@ -58,11 +58,22 @@ <h1>The Regis Altare Birthday Project 2023</h1>
           Github</a
         >
       </footer>
+      <div
```

Comment:
you probably didn't apply my recent changes, but just to be sure I moved the dialog code to a separate file